src/components/Compare/CompareStripedRows.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid } from "@trussworks/react-uswds";
+
+// TODO: display checkbox for boolean values
+
+function CompareStripedRows({
+  rows,
+}: {
+  rows: {
+    label: string;
+    compareA: string | boolean;
+    compareB: string | boolean;
+  }[];
+}) {
+  return (
+    <div className="border-top margin-top-05 margin-bottom-3">
+      {rows.map((row, idx) => (
+        <Grid
```

Comment:
are there accessibility implications to not using actual table HTML elements but displaying tabular data?

Reply:
it seems like the benefit of html tables is that screenreaders can read the corresponding header rows for each cell, which makes more sense for wide tables that people can quickly get lost in - since we only have two columns, reading the header with every cell doesn't feel as necessary. i was poking around some of the diff compare examples that adam pulled together, and i didn't see any that used actual html table elements, so now i feel more ok about going down this path